.github/workflows/e2e.yaml

```diff
@@ -24,125 +24,97 @@ on:
     tags:
       - 'v*'
 
+env:
+  SKIP_TEST: true
+  SW_TEST_ID: ${GITHUB_RUN_ID}
+  SW_LOG_DIR: ${HOME}/logs/${GITHUB_RUN_ID}/
+
 jobs:
-  Single:
+  Checkstyle:
+    name: Check Codes Style
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      # In the checkout@v2, it doesn't support git submodule. Execute the commands manually.
-      - name: checkout submodules
+      - name: Checkout Submodules
         shell: bash
         run: |
           git submodule sync --recursive
           git -c protocol.version=2 submodule update --init --force --recursive --depth=1
-      - name: Set environment
-        run: export MAVEN_OPTS='-Dmaven.repo.local=~/.m2/repository -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -XX:-UseGCOverheadLimit -Xmx3g'
-      - name: Compile & Install Test Codes
-        run: |
-          ./mvnw --batch-mode checkstyle:check apache-rat:check
-          ./mvnw --batch-mode -Dcheckstyle.skip -Drat.skip -T2 -Dmaven.compile.fork -Dmaven.compiler.maxmem=3072 -DskipTests clean install
-          ./mvnw --batch-mode -f test/e2e/pom.xml -pl e2e-base clean install
-      - name: Single Node Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-single-service
-      - name: Single Node Tests(InfluxDB/JDK8)
-        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-influxdb
-      - name: Single Node Tests(MySQL/JDK8)
-        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-mysql
-      - name: Single Node Tests(JDK9)
-        run: export E2E_VERSION=jdk9-1.5 && bash -x test/e2e/run.sh e2e-single-service
-      - name: Single Node Tests(JDK11)
-        run: export E2E_VERSION=jdk11-1.5 && bash -x test/e2e/run.sh e2e-single-service
-      - name: Single Node Tests(JDK12)
-        run: export E2E_VERSION=jdk12-1.5 && bash -x test/e2e/run.sh e2e-single-service
-      - name: Agent Reboot Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-agent-reboot
-      - name: Agent Auth Tests(JDK8)
-        run: export E2E_VERSION=jdk8-1.5 && bash -x test/e2e/run.sh e2e-agent-auth
+      - name: Checkstyle
+        run: ./mvnw --batch-mode -f test/e2e/pom.xml -DskipTests checkstyle:check
 
-  Cluster:
+  FeaturesGroup1:
```

Comment:
Once this name changed, we are facing the awkward situation. The PR can't be merged as we set that by the INFRA team. Could we consider to change it back?

Reply:
> Once this name changed, we are facing the awkward situation. The PR can't be merged as we set that by the INFRA team. Could we consider to change it back?

I begin to use matrix builds, so even I change back to `Single` or `Cluster`, the names are still not the same as before, besides, the original name is rather vague, and we could make it more clear, if you agree, I can open a ticket to ask the infra team for the help, to change the required statuses

Reply:
I am OK to change. Just we need
1. A consensus  about the task lists 
1. Your PR passed and ready to be merged immediately. 

We could request the INFRA. Because otherwise, we can’t merge any PR if INFRA changed that before this is ready.